src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,156 @@
+
+  const { slug } = useParams();
+
+  const listOfNames = useMemo(
+    () => people.map(person => person.name),
+    [people],
+  );
+
+  const findSlugOfChoosedPerson = (choosedPerson: string) => {
+    return people.find(person => person.name === choosedPerson)?.slug;
+  };
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && (
+            <Loader />
+          )}
+
+          {errorMessage && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {!people.length && !isLoading && !errorMessage && (
+            <p data-cy="noPeopleMessage">
+              There are no people on the server
+            </p>
+          )}
+
+          {people.length > 0 && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people.map(person => (
+                  <tr
+                    key={person.slug}
+                    data-cy="person"
+                    className={classNames({
+                      'has-background-warning': slug === person.slug,
+                    })}
+                  >
+                    <td>
+                      <Link
+                        relative="path"
+                        to={`../${person.slug}`}
+                        className={classNames({
+                          'has-text-danger': person.sex === 'f',
+                        })}
+                      >
+                        {person.name}
+                      </Link>
+                    </td>
+
+                    <td>{person.sex}</td>
+                    <td>{person.born}</td>
+                    <td>{person.died}</td>
+
+                    {person.motherName
+                      && listOfNames.includes(person.motherName)
+                      && (
+                        <td>
+                          <Link
+                            to={`../${findSlugOfChoosedPerson(person.motherName)}`}
+                            className="has-text-danger"
+                          >
+                            {person.motherName}
+                          </Link>
+                        </td>
+                      )}
+
+                    {person.motherName
+                      && !listOfNames.includes(person.motherName)
+                      && (
+                        <td>
+                          {person.motherName}
+                        </td>
+                      )}
+
```

Comment:
try to use ternary operator, it will make your code shorter

```
{person.motherName
  && listOfNames.includes(person.motherName)
  && ? (
    <td>
      <Link 
        to={`../${findSlugOfChoosedPerson(person.motherName)}`}
        className="has-text-danger">
          {person.motherName}
      </Link>
    </td>
  ) : (
     <td>{person.motherName || '-'}</td>
  )}
 ````